src/components/NavigationPanel.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  FaFacebookF, FaTwitter, FaLinkedinIn, FaGithub,
+} from 'react-icons/fa';
+import LogoutButton from './session/LogoutButton';
+
+const NavigationPanel = () => {
+  const [showLinks, setShowLinks] = useState(false);
+
+  return (
+    <section className="nav-section">
+      <nav>
+        <div className="header">
+          <p className="ms-1">Logo</p>
+          <button
+            type="button"
+            data-testid="hamburger-button"
+            className={`hamburger ${showLinks ? 'active' : ''}`}
+            onClick={() => setShowLinks(!showLinks)}
+          >
+            <span className="bar" />
+            <span className="bar" />
+            <span className="bar" />
+          </button>
+        </div>
+
+        {showLinks && (
+          <div className="nav-links" data-testid="links-container">
+            <Link to="/tables">Tables</Link>
+            <Link to="/AddTable">Add Table</Link>
+            <Link to="/DeleteTable">Delete Table</Link>
+            <Link to="/reservation-form">Make Reservation</Link>
+            <Link to="/Myresercvations">My Reservations</Link>
+            <LogoutButton />
+            <Link to="/homepage" onClick={() => setShowLinks(false)}>
+              Home
+            </Link>
+          </div>
+        )}
```

Comment:
- [ ] On your mobile view, the very first link "Tables" leads to a blank page on /tables:

![image](https://github.com/george827/Book-reservation-Front_End/assets/57421823/0b2e63f3-d723-4200-ad2f-64b852ff375a)


![image](https://github.com/george827/Book-reservation-Front_End/assets/57421823/060a4282-7cb3-4197-878d-666502ba7cb0)

This is because there is no route or component for `/tables`.

This may have been introduced when the navbar was implemented on all the components.

Since you already have a "Home" link there, kindly just remove the link `"/tables"` in line 30 and the problem will be solved. 💪